### DISCUSSION TITLE
Data Set

### DISCUSSION THREAD

**Discussion by timvahlbrock** [Oeffis/Oeffis#12] [Ideas]
The creation of a dataset, that contains info on which train was delayed where at a certain point of time might be pretty valueable in itself - for us and also for other projects.

**Comment by WConen** [comment]
One potential source for data: https://www.zugfinder.net/de/start (seemingly no API, but some interesting usage ideas, https://www.zugfinder.net/de/technik, including an experimental AI to predict delays). 

Some data sources have been mentioned in the Exposé, perhaps we could collect them (and additional one) here in the discussion and consolidate it later into an "epic" ticket (I guess, that was the plan anyway ;)
